### PR TITLE
getxbook: test added, build fixed

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

File: Library/Formula/getxbook.rb [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+  option "with-gui", "Build the GUI"
+
+  depends_on "tcl-tk" if build.with? "gui"
 
   def install
-    system "make", "CC=#{ENV.cc}", "PREFIX=#{prefix}", "install"
+    args = %W[CC=#{ENV.cc} PREFIX=#{prefix}]
+
+    system "make", *args
+    system "make", *args, "install" if build.with?("gui")
```
Comment (mikemcquaid) [inline Library/Formula/getxbook.rb]: `*args` must be the last argument on 10.8.

File: Library/Formula/getxbook.rb [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+    bin.install "getgbook", "getabook", "getbnbook"
+  end
+
+  test do
+    %w[getgbook getabook getbnbook].each do |tool|
+      assert_match "#{tool} #{version}", shell_output("#{bin}/#{tool}", 1)
+    end
```
Comment (mikemcquaid) [inline Library/Formula/getxbook.rb]: Just run one tool.

File: Library/Formula/getxbook.rb (hunk 1 above)
Comment (bfontaine) [inline Library/Formula/getxbook.rb]: @mikemcquaid Fixed.

File: Library/Formula/getxbook.rb [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
+
+  depends_on "tcl-tk" if build.with? "gui"
 
   def install
-    system "make", "CC=#{ENV.cc}", "PREFIX=#{prefix}", "install"
+    args = %W[CC=#{ENV.cc} PREFIX=#{prefix}]
+    args << "install" if build.with?("gui")
+
+    system "make", *args
+    bin.install "getgbook", "getabook", "getbnbook"
```
Comment (xu-cheng) [inline Library/Formula/getxbook.rb]: So if I install `with-gui`, will I get two copies of binary, one in `bin`, one in app bundle? If that's the case, using symlink may be a better idea.

Comment (bfontaine) [inline Library/Formula/getxbook.rb]: There’s no app bundle, only a `getxbookgui` binary.

Comment (xu-cheng) [inline Library/Formula/getxbook.rb]: Well, great job for your work. :+1: :tada:

Comment (bfontaine) [inline Library/Formula/getxbook.rb]: Thanks :smile:

Comment (mikemcquaid) [inline Library/Formula/getxbook.rb]: You two are my heroes :metal:

**Pull request closed without merging**